source/renderer/app/utils/formatters.js

```diff
@@ -7,9 +7,42 @@ import {
 
 export const formattedWalletAmount = (
   amount: BigNumber,
-  withCurrency: boolean = true
+  withCurrency: boolean = true,
+  long: boolean = false
 ) => {
-  let formattedAmount = amount.toFormat(DECIMAL_PLACES_IN_ADA);
+  let formattedAmount = '';
+  if (long) {
+    formattedAmount = amount.toFormat(DECIMAL_PLACES_IN_ADA);
+  } else if (amount.isZero()) {
+    formattedAmount = '0';
+  } else if (amount.lessThan(0.000001)) {
+    formattedAmount = '< 0.000001';
```

Comment:
not sure it's applicable to ADA currency, but just FYI it's not needed here :)

Reply:
Goot catch @a-rukin - we don't need this case @DeeJayElly - it was an ETC specific case. Please remove it 🙏

Reply:
Fixed